### DISCUSSION TITLE
输入法引擎支持多个预编辑同时搜索

### DISCUSSION THREAD

**Discussion by xingwozhonghua126** [fcitx/fcitx5#356] [Q&A]
因为平板9宫格键盘需要支持多预编辑搜索，并返回候选词。同时9宫格也可以在pc上对应小键盘。

**Comment by wengxt** [comment]
你得具体解释一下是什么意思，我觉得九宫格和普通输入法至少在预编辑上没什么区别

**Reply by xingwozhonghua126** [reply]
当我按下九宫格输入法中的5是，上面的的预编辑显示的是l，然而候选词不是根据l索引的，是根据左面若干预编辑综合索引的。左面的可选预编辑有若干，选择左面的预编辑才是单一索引。

**Reply by xingwozhonghua126** [reply]
![Screenshot_20211012_124052](https://user-images.githubusercontent.com/18661883/136894072-ecb3a6f0-6747-498c-b226-266644af7785.jpg)

**Reply by xingwozhonghua126** [reply]
9宫格的按键模式最好和小键盘类似，这样PC也能方便操作。

![v2-165500bfddb15d512db3c6810a0d9084_r](https://user-images.githubusercontent.com/18661883/136898520-d41b2069-1819-4c45-9f19-603847b9f7ea.jpg)

**Reply by wengxt** [reply]
我的意思是这个和 fcitx 没什么关系。输入法算法的部分和 fcitx 都没什么关系。怎么 libime 支持才是需要考虑的

**Reply by xingwozhonghua126** [reply]
是的，应该在libime处理，我把它移动到libime里面，关掉fcitx5的这个？

**Reply by xingwozhonghua126** [reply]
https://github.com/fcitx/libime/issues/28
